src/utilities/fe_mesh.jl

```diff
@@ -0,0 +1,162 @@
+# Typ definitions
+type FEBoundary # Tänkte typ parametrisera detta på nåt sätt så att man har FEboundary i 2D, 1D, och 0D
```

Comment:
So then I would want to gather all these boundaries into the `FEMesh` field `boundary` as a vector of vectors, so that the first vector contains all the 1D boundaries and the second all the 0D boundaries.

Example:

```jl
fe_m = get2Dmesh(...)
fe_m.boundary[1][1] # This gives the first 1D boundary
fe_m.boundary[1][2] # This gives the second 1D boundary
fe_m.boundary[0][1] # This gives the first 0D boundary
```

But how do I define that in the `FEMesh` type?

```jl
type FEMesh
    ...
    boundary::Vector{Vector{FEBoundary}}
end
```

But with the definition of `FEBoundary` with

```jl
type FEBoundary{T}
    topology::Array{Int}
end
```
That means that just `FEBoundary` is abstract, right? So then I don't want it like above in the type definition of `FEMesh`? :confused: